__tests__/api.test.js

```diff
@@ -48,6 +48,40 @@ describe("/api/topics", () => {
 	});
 });
 
+describe("/api/articles", () => {
+	test("GET STATUS 200: should return an array of articles objects with their respective properties.", () => {
+		return request(app)
+			.get("/api/articles")
+			.expect(200)
+			.then((articles) => {
+				expect(articles.body.articles.length).toBe(13);
+				expect(articles.body.articles[0]).toMatchObject({
```

Comment:
You could make this more robust by checking that every article looks as it should, rather than just the first article in the array. You can do this by using `.toMatchObject` inside a `forEach`, and using 
`.toMatchObject({ author: expect.any(String), topic:  expect.any(String) });`
and so on. So although you're not checking the specific value of each property, you're checking they are the data type that they should be (string, number etc)